filters/awssigner/internal/credentials.go

```diff
@@ -0,0 +1,36 @@
+package awssigner
+
+import (
+	"path"
+	"time"
+)
+
+// A Credentials is the AWS credentials value for individual credential fields.
+type Credentials struct {
+	// AWS Access key ID
+	AccessKeyID string
+
+	// AWS Secret Access Key
+	SecretAccessKey string
```

Comment:
Go doc of the type and all members are wrong here.
Go doc wants to have comments for everything exported (func, types, members, ..) like this:

```
// Foo ....
type Foo struct {
    // Bar ...
    Bar ...
}

// Qux ...
func Qux() ...

// Baz ...
func (m *M) Baz() ...
```

Please fix the Go doc style comments.

Reply:
> Go doc of the type and all members are wrong here. Go doc wants to have comments for everything exported (func, types, members, ..) like this:
> 
> ```
> // Foo ....
> type Foo struct {
>     // Bar ...
>     Bar ...
> }
> 
> // Qux ...
> func Qux() ...
> 
> // Baz ...
> func (m *M) Baz() ...
> ```
> 
> Please fix the Go doc style comments.

changed, thanks